### ISSUE TITLE
Warning when compiling mix_audit with older Elixir version

### ISSUE THREAD

**Issue by okalouti** [mirego/mix_audit#3]
I am using an older version of Elixir (`1.8.0`) and I get the following warning when compiling the `mix_audit` dependency. 

```
warning: the dependency :mix_audit requires Elixir "~> 1.9" but you are running on v1.8.0
```

The command `mix deps.audit` runs fine after that, but I wanted to ask if it does cause an actual issue to be running a version of Elixir older than 1.9? 

Thanks in advance for clarifying!

**Comment by remi** [comment]
Hi Omar,

No, there’s no reason `mix_audit` shouldn’t run correctly on Elixir 1.8. I’m working on a fix that supports it.

Thank you!

**Comment by remi** [comment]
@okalouti I just released `mix_audit` 0.1.3 which should not generate a warning on Elixir 1.8 😄

**Comment by okalouti** [comment]
Thank you @remi !
